spark3/src/main/java/org/apache/iceberg/spark/procedures/ExpireSnapshotsProcedure.java

```diff
@@ -96,6 +75,36 @@ public StructType outputType() {
     return OUTPUT_TYPE;
   }
 
+  @Override
+  public InternalRow[] call(InternalRow args) {
+    String namespace = args.getString(0);
+    String tableName = args.getString(1);
+    Long olderThanMillis = args.isNullAt(2) ? null : DateTimeUtils.toMillis(args.getLong(2));
+    Integer retainLastNum = args.isNullAt(3) ? null : args.getInt(3);
+
+    return modifyIcebergTable(namespace, tableName, table -> {
+      Actions actions = Actions.forTable(table);
+
+      ExpireSnapshotsAction action = actions.expireSnapshots();
+
+      if (olderThanMillis != null) {
+        action.expireOlderThan(olderThanMillis);
```

Comment:
So we are going with the action default expire time? Just checking because I thougt some folks wanted that changed

Reply:
The action delegates to `ExpireSnapshots` table API that, in turn, respects the default table props we added recently.

Reply:
Ah! Excellent, I forgot about that